core/identity/src/service/identity.rs

```diff
@@ -32,10 +49,36 @@ fn to_info(default_key: &NodeId, key: &IdentityKey) -> model::IdentityInfo {
     }
 }
 
+fn send_event(s: Ref<Subscription>, event: model::event::Event) -> impl Future<Output = ()> {
+    let subscriptions: Vec<String> = s.subscriptions.clone();
+    log::debug!("sending event: {:?} to {:?}", event, subscriptions);
+
+    async move {
+        for endpoint in subscriptions {
```

Comment:
There should be a separate spawn call for every subscription. Otherwise, one subscriber can effectively block others.